src/components/expanded-state/custom-gas/FeesPanel.js

```diff
@@ -199,19 +202,15 @@ export default function FeesPanel({
   );
 
   const { maxFee, currentBaseFee, maxBaseFee, maxPriorityFee } = useMemo(() => {
-    const maxFee = selectedGasFee?.gasFee?.maxFee?.native?.value?.display || 0;
-    const currentBaseFee = currentBlockParams?.baseFeePerGas?.gwei || 0;
-
+    const maxFee = selectedGasFee?.gasFee?.maxFee?.native?.value?.display;
+    const currentBaseFee = currentBlockParams?.baseFeePerGas?.gwei;
     const maxBaseFee = selectedOptionIsCustom
       ? customMaxBaseFee
-      : toFixedDecimals(
-          selectedGasFee?.gasFeeParams?.maxFeePerGas?.gwei || 0,
-          0
-        );
+      : toFixedDecimals(selectedGasFee?.gasFeeParams?.maxFeePerGas?.gwei, 0);
```

Comment:
if `selectedGasFee?.gasFeeParams?.maxFeePerGas?.gwei` is undefined, then the `toFixedDecimals` will return `NaN`